plugins/mtg.py

```diff
@@ -78,27 +78,42 @@ def mtg(inp, say=None):
         '{B}'   : 'B' , 
         '{R}'   : 'R' , 
         '{G}'   : 'G' , 
-        '{W/P}' : u'\u03A6' , 
-        '{U/P}' : u'\u03A6' , 
-        '{B/P}' : u'\u03A6' , 
-        '{R/P}' : u'\u03A6' , 
-        '{G/P}' : u'\u03A6' ,
+        '{W/P}' : u'\u03D5' , 
+        '{U/P}' : u'\u03D5' , 
+        '{B/P}' : u'\u03D5' , 
+        '{R/P}' : u'\u03D5' , 
+        '{G/P}' : u'\u03D5' ,
+        '{X}'   : 'X' ,
         '\n' : ' ' ,
     }
     results = {
         "name": card["name"],
         "types": ", ".join(t.capitalize() for t in card["types"]),
         "cost": card["cost"],
         "text": card["text"],
+        "power": card["power"] if card.get("power") is not None else None,
+        "toughness": card["toughness"] if card.get("toughness") is not None else None,
         "multiverse_id": card["editions"][valid_edition]["multiverse_id"],
     }
+    results["supertypes"] = ", ".join(t.capitalize() for t in card["supertypes"]) if card.get("supertypes") is not None else None
+    results["subtypes"] = ", ".join(t.capitalize() for t in card["subtypes"]) if card.get("subtypes") is not None else None
+
     for str,rep in symbols.items():
         new_text = results["text"].replace(str,rep)
         new_cost = results["cost"].replace(str,rep)
         results["text"] = new_text
         results["cost"] = new_cost
-
-    return u"{name} - {types} - {cost} | {text} | http://gatherer.wizards.com/Pages/Card/Details.aspx?multiverseid={multiverse_id}".format(**results)
+    
+    response = [u"{name} -".format(**results)]
+    response.append(u"{supertypes}".format(**results)) if results["supertypes"] is not None else None
```

Comment:
if "supertypes" in card:
        response.append(", ".join(t.capitalize() for t in card["supertypes"]))